frontends/election-manager/src/screens/write_ins_transcription_screen.tsx

```diff
@@ -101,10 +103,10 @@ function WriteInImage({
       src={imageUrl}
       alt="write-in area"
       crop={{
-        x: bounds.x,
-        y: bounds.y - bounds.height * margin,
-        width: bounds.width,
-        height: bounds.height * (1 + 2 * margin),
+        x: bounds.x * IMAGE_SCALE,
+        y: bounds.y * IMAGE_SCALE - bounds.height * margin,
```

Comment:
```suggestion
        y: (bounds.y - bounds.height) * IMAGE_SCALE * margin,
```

`bounds.height` should be scaled too, right?

Reply:
yeah probably, should margin be multiplied by both things or just the height?